libs/blocks/table/table.css

```
@@ -664,6 +664,26 @@
     margin: 0 30px;
   }
 
+  .table .left-round {
+    border-top-left-radius: var(--border-radius);
+    border-top-right-radius: 0;
+  }
+  
+  .table .right-round {
+    border-top-right-radius: var(--border-radius);
+    border-top-left-radius: 0;
+  }
+  
+  [dir="rtl"] .table .left-round {
+    border-top-right-radius: var(--border-radius);
+    border-top-left-radius: 0;
+  }
```

Why not combine these, since they declare the same thing?
```suggestion
  .table .right-round,
  [dir="rtl"] .table .left-round {
    border-top-right-radius: var(--border-radius);
    border-top-left-radius: 0;
  }
```
Same for the other two new declarations.